docs/Gemfile

```diff
@@ -7,21 +7,23 @@ source "https://rubygems.org"
 #     bundle exec jekyll serve
 #
 # This will help ensure the proper Jekyll version is running.
-# Happy Jekylling!
-gem "jekyll", "~> 3.6.0"
+gem "jekyll", "3.7.3"
 
 # This is the default theme for new Jekyll sites. You may change this to anything you like.
-gem "minima", "2.1.1"
+gem "minima", "2.4.1"
+gem "jekyll-theme-cayman"
 
 # If you want to use GitHub Pages, remove the "gem "jekyll"" above and
```

Comment:
doesn't this say you either have the 'github-pages' gem, or specify the 'jekyll' gem?

i guess the github-pages gem includes a specific version of the jekyll gem? maybe that's what bit us in the first place, that the github-pages used a different version than we required here?